src/components/sykmeldte/Sykmeldte.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { Ref } from 'react';
+import { NonEmptyArray } from 'remeda/dist/commonjs/_types';
+import { Cell } from '@navikt/ds-react';
+
+import { PreviewSykmeldtFragment } from '../../graphql/queries/graphql.generated';
+import ExpandableSykmeldtPanel from '../shared/SykmeldtPanel/ExpandableSykmeldtPanel';
+
+import { useExpanded, useExpandSykmeldte } from './useExpandSykmeldte';
+import OrgHeading from './OrgHeading';
+
+interface Props {
+    sykmeldteGrouped: [string, NonEmptyArray<PreviewSykmeldtFragment>][];
```

Comment:
hmm, jeg angrer litt på at jeg brukte `'default'`, det er litt svakt typet. Hva om vi gjør denne til `sykmeldteGrouped: [string | null, NonEmptyArray<PreviewSykmeldtFragment>][];`, sjekker på null isteden for default nede i JSXen, og gjør groupingen om til `groupBy(list, (it) => (showOrgHeading ? it.orgnavn : null)`?

Tror det burde være litt bedre typing :)

Reply:
Typing av key i `groupBy` er `declare type PropertyKey = string | number | symbol`; 
Så då tenker eg at vi berre beheld 'default' her.